testsrc/org/mozilla/javascript/tests/Test262SuiteTest.java

```diff
@@ -436,6 +438,50 @@ public static void tearDownClass() {
     private final Test262Case testCase;
     private final boolean markedAsFailing;
 
+    /** @see https://github.com/tc39/test262/blob/main/INTERPRETING.md#host-defined-functions */
+    private class $262 {
+        private ScriptableObject scope;
+
+        public $262(ScriptableObject scope) {
+            this.scope = scope;
+
+            scope.put("$262", scope, this);
+            scope.setAttributes("$262", ScriptableObject.DONTENUM);
+        };
+
+        @JSFunction
+        public void gc() {
+            System.gc();
+        }
+
+        @JSFunction
+        public Object evalScript(String source) {
+            return Context.getCurrentContext()
+                    .evaluateString(this.scope, source, "<evalScript>", 1, null);
+        }
+
+        @JSGetter
+        public Object getGlobal() {
+            return this.scope;
+        }
+
+        @JSFunction
+        public $262 createRealm() {
+            return new $262(Context.getCurrentContext().initSafeStandardObjects());
+        }
+
+        @JSFunction
+        public void detachArrayBuffer() {
+            throw new UnsupportedOperationException(
+                    "$262.detachArrayBuffer() method not yet implemented");
+        }
+
+        @JSGetter
+        public Object getAgent() {
+            throw new UnsupportedOperationException("$262.agent property not yet implemented");
+        }
+    };
```

Comment:
;

Reply:
Did some tests but there is no effect, maybe we need more realm support

Reply:
Lookin'... This is code I wrote quite a while ago and at the time adding the $262 stuff made some tests pass, but now they dont...

Reply:
Pushed a fix for the $262 object not having much of a difference

Also updated the test262.properties now